File: 백준/Bronze/2739. 구구단/구구단.go
package main

import (
	"bufio"
	"fmt"
	"os"
)

func main() {
	reader := bufio.NewReader(os.Stdin)
	writer := bufio.NewWriter(os.Stdout)
	defer writer.Flush()
	var n int
	express := "%d * %d = %d\n"
	fmt.Fscan(reader, &n)
	for i := 1; i <= 9; i++ {
		fmt.Fprintf(writer, express, n, i, i*n)
	}

}

File: 백준/Bronze/2739. 구구단/구구단.rs
use std::io::{self, BufRead, BufReader, Write};

fn main() {
    let mut reader = BufReader::new(io::stdin().lock());
    io::stdout().flush().unwrap();

    let mut input = String::new();
    reader.read_line(&mut input).expect("");

    let num: i32 = input.trim().parse().expect("");

    (1..10).for_each(|i| println!("{} * {} = {}", num, i, num * i));
}
